pkg/aws/ocpgwdeployer.go

```diff
@@ -157,6 +164,10 @@ func (d *ocpGatewayDeployer) validateDeployPrerequisites(vpcID string, input api
 		return utilerrors.NewAggregate(errs)
 	}
 
+	if d.useLoadBalancer {
+		return nil
```

Comment:
Actually for LB the `validateCreateSecGroup` call is not needed, so the function needs to be restructured